src/Components/MainContent.js

```diff
@@ -0,0 +1,59 @@
+// Main Content 
+const MainContent = (prop) => {
+  const artObject = prop.art;
+
+  // destructure prop
+  const {artistDisplayName, artistDisplayBio, department, dimensions, medium, primaryImageSmall, repository, title, GalleryNumber, creditLine, tags} = prop.art;
+
+  // save function
+  const handleSave = prop.save;
+
+  return ( 
+    <main className="wrapper">
+      {
+        artObject.objectID &&
+        (
+        <>
+        <p>An art piece for {prop.search}</p>
+        <div className="artDisplay">
+          <div className="imgContainer">
+            <img src={primaryImageSmall} alt={`${title} by ${artistDisplayName}`}  />
+          </div>
+
+          <div className="description">  
+            <button className="save-btn" onClick={() => handleSave(primaryImageSmall, title)}>Save</button>
+            <h2>{title}</h2>       
```

Comment:
nicely done, the info displayed for each art piece is like what your would see on a museum description plaque